app/javascript/pages/IndividualEvents/index.js

```diff
@@ -97,6 +97,9 @@ const validate = values => {
   if (!values.duration) {
     errors.duration = 'is required'
   }
+  if (values.duration > 8 * 60) {
```

Comment:
where did the 8 hours come from? Before we merge this I would like to confirm with everyone that 8 hours is the max.

Reply:
@azaiszen from CSR Committee suggests that 24 hours is a reasonable limit.